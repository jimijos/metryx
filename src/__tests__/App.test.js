import React from "react";
import { render, cleanup } from "@testing-library/react";
import Loding from "../components/Loading";

afterEach(cleanup);

it("should take a snapshot of initial Loading Component", () => {
  const { asFragment } = render(<Loding />);

  expect(asFragment(<Loding />)).toMatchSnapshot();
});
