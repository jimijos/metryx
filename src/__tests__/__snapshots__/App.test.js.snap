// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot of initial Loading Component 1`] = `
<DocumentFragment>
  <div
    style="width: 100vw; height: 100vh; position: absolute; background: rgba(10, 12, 10, 0.9); z-index: 10;"
  >
    <span
      class="css-1rc3du5"
    />
  </div>
</DocumentFragment>
`;
